x/incentive/types/claims.go

```diff
@@ -314,6 +315,71 @@ func (cs DelegatorClaims) Validate() error {
 	return nil
 }
 
+// SwapClaim stores the swap rewards that can be claimed by owner
+type SwapClaim struct {
+	BaseMultiClaim `json:"base_claim" yaml:"base_claim"`
+	RewardIndexes  MultiRewardIndexes `json:"reward_indexes" yaml:"reward_indexes"`
+}
+
+// NewSwapClaim returns a new SwapClaim
+func NewSwapClaim(owner sdk.AccAddress, rewards sdk.Coins, rewardIndexes MultiRewardIndexes) SwapClaim {
+	return SwapClaim{
+		BaseMultiClaim: BaseMultiClaim{
+			Owner:  owner,
+			Reward: rewards,
+		},
+		RewardIndexes: rewardIndexes,
+	}
+}
+
+// GetType returns the claim's type
+func (c SwapClaim) GetType() string { return SwapClaimType }
+
+// GetReward returns the claim's reward coin
+func (c SwapClaim) GetReward() sdk.Coins { return c.Reward }
+
+// GetOwner returns the claim's owner
+func (c SwapClaim) GetOwner() sdk.AccAddress { return c.Owner }
+
+// Validate performs a basic check of a HardLiquidityProviderClaim fields
```

Comment:
HardLiquidityProviderClaim -> SwapClaim

Reply:
fixed in https://github.com/Kava-Labs/kava/pull/950/commits/c031697fc1b8981a40c2629eaf9be6b9138455c3